src/display/text_layer.js

```diff
@@ -20,8 +20,10 @@ import { CustomStyle, getDefaultSetting } from './dom_utils';
  * Text layer render parameters.
  *
  * @typedef {Object} TextLayerRenderParameters
- * @property {TextContent} textContent - Text content to render (the object is
- *   returned by the page's getTextContent() method).
+ * @property {TextContent} textContent - (optional) Text content to render
+ *   (the object is returned by the page's getTextContent() method).
+ * @property {ReadableStream} textContentStream - (optional) Text Content stream
+ *   to render(the stream is returned by the page's streamTextContent() method).
```

Comment:
Nit: Missing space between `render` and `(`.

Also, there needs to be a JSDoc comment for `textContentItemsStr` as well (similar to `textDivs` below).